content/en/serverless/installation/java.md

```diff
@@ -43,81 +137,59 @@ Datadog offers many different ways to enable instrumentation for your serverless
     RUN wget -O /opt/dd-java-agent.jar https://dtdg.co/latest-java-tracer
     ```
 
-3. Install the Datadog Lambda library
-    
-    If you are using Maven, include the following dependency in your `pom.xml`, and replace `VERSION` with the latest release (omitting the preceeding `v`): ![Maven Cental][2]:
-
-    ```xml
-    <dependency>
-      <groupId>com.datadoghq</groupId>
-      <artifactId>datadog-lambda-java</artifactId>
-      <version>VERSION</version>
-    </dependency>
-    ```
-    
-    If you are using Gradle, include the following dependency in your `build.gradle`, and replace `VERSION` with the latest release (omitting the preceeding `v`): ![Maven Cental][2]:
-
-    ```groovy
-    dependencies {
-      implementation 'com.datadoghq:datadog-lambda-java:VERSION'
-    }
-    ```
-
-4. Set the required environment variables
+3. Set the required environment variables
 
     - Set `AWS_LAMBDA_EXEC_WRAPPER` to `/opt/datadog_wrapper`.
-    - Set `DD_TRACE_ENABLED` to `true`
     - Set `DD_SITE` to {{< region-param key="dd_site" code="true" >}} (ensure the correct SITE is selected on the right).
     - Set `DD_API_KEY_SECRET_ARN` to the ARN of the AWS secret where your [Datadog API key][3] is securely stored. The key needs to be stored as a plaintext string (not a JSON blob). The `secretsmanager:GetSecretValue` permission is required. For quick testing, you can use `DD_API_KEY` instead and set the Datadog API key in plaintext.
 
 [1]: https://gallery.ecr.aws/datadog/lambda-extension
-[2]: https://img.shields.io/maven-central/v/com.datadoghq/datadog-lambda-java
-[3]: https://app.datadoghq.com/organization-settings/api-keys
+[2]: https://app.datadoghq.com/organization-settings/api-keys
```

Comment:
Need to update the link index, `[Datadog API key][3]` is now broken.

Reply:
thanks, good catch